Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -9,7 +9,7 @@ import BraintreeCore
 import BraintreeDataCollector
 #endif
 
-@objcMembers public class BTPayPalClient: NSObject {
+@objc public class BTPayPalClient: NSObject {
```

Comment:
What is the difference between these two at class level?

Reply:
From the Swift docs:

[`@objcMembers`](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#objcMembers):
> Apply this attribute to a class declaration, to implicitly apply the objc attribute to all Objective-C compatible members of the class, its extensions, its subclasses, and all of the extensions of its subclasses.

[`@objc`](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#objc):
> Apply this attribute to any declaration that can be represented in Objective-C — for example, nonnested classes, protocols, nongeneric enumerations (constrained to integer raw-value types), properties and methods (including getters and setters) of classes, protocols and optional members of a protocol, initializers, and subscripts. The objc attribute tells the compiler that a declaration is available to use in Objective-C code.

We do not want to expose both the async and completion methods to Objective-C (discussed [here](https://github.com/braintree/braintree_ios/pull/975#discussion_r1138484648)) so opted for the explicit `@objc` in the clients.